package.json

```diff
@@ -15,6 +15,12 @@
     "prepare": "husky install",
     "gen-pdf": "/bin/bash ./script/setup-dev.sh pdf-setup"
   },
+  "files": [
```

Comment:
What is `files` config doing here?

Reply:
This array specify to npm which files to keep during the build process of the package.

See an example here:
https://github.com/proofcarryingdata/zupass/blob/bf91f9b3b63ef7326fe16ce0318cee0847fcc4b0/packages/semaphore-group-pcd/package.json#L7